<a href="https://colab.research.google.com/github/ragedharb16/sales-predictions/blob/main/projects_2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd

In [2]:
df=pd.read_csv("/content/heart.csv")
df

Age Sex ChestPainType  RestingBP  Cholesterol  FastingBS RestingECG  \
0     40   M           ATA        140          289          0     Normal   
1     49   F           NAP        160          180          0     Normal   
2     37   M           ATA        130          283          0         ST   
3     48   F           ASY        138          214          0     Normal   
4     54   M           NAP        150          195          0     Normal   
..   ...  ..           ...        ...          ...        ...        ...   
913   45   M            TA        110          264          0     Normal   
914   68   M           ASY        144          193          1     Normal   
915   57   M           ASY        130          131          0     Normal   
916   57   F           ATA        130          236          0        LVH   
917   38   M           NAP        138          175          0     Normal   

     MaxHR ExerciseAngina  Oldpeak ST_Slope  HeartDisease  
0      172              N      0.0       Up             0  
1      156              N      1.0     Flat             1  
2       98              N      0.0       Up             0  
3      108              Y      1.5     Flat             1  
4      122              N      0.0       Up             0  
..     ...            ...      ...      ...           ...  
913    132              N      1.2     Flat             1  
914    141              N      3.4     Flat             1  
915    115              Y      1.2     Flat             1  
916    174              N      0.0     Flat             1  
917    173              N      0.0       Up             0  

[918 rows x 12 columns]


##option A

1) Source of data


https://www.kaggle.com/fedesoriano/heart-failure-prediction


This dataset was created by combining different datasets already available independently but not combined before. In this dataset, 5 heart datasets are combined over 11 common features which makes it the largest heart disease dataset available so far for research purposes. The five datasets used for its curation are:

Cleveland: 303 observations
Hungarian: 294 observations
Switzerland: 123 observations
Long Beach VA: 200 observations
Stalog (Heart) Data Set: 270 observations

2) Brief description of data


Cardiovascular diseases (CVDs) are the number 1 cause of death globally, taking an estimated 17.9 million lives each year, which accounts for 31% of all deaths worldwide. Four out of 5CVD deaths are due to heart attacks and strokes, and one-third of these deaths occur prematurely in people under 70 years of age. Heart failure is a common event caused by CVDs and this dataset contains 11 features that can be used to predict a possible heart disease.

People with cardiovascular disease or who are at high cardiovascular risk (due to the presence of one or more risk factors such as hypertension, diabetes, hyperlipidaemia or already established disease) need early detection and management wherein a machine learning model can be of great help.


3) What is the target?


HeartDisease: output class [1: heart disease, 0: Normal]

4) Is this a classification or regression problem?

classifications

5) How many features?


11 features and 1 target

6) How many rows of data.

918 rows



7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

there is challenges in cleaning visualitation and exploring this data, also and the predicctions models that we are going to use



In [3]:
df1=pd.read_csv("/content/diabetesprediction.csv")
df1.shape

(768, 8)

##option B

1) Source of data


https://www.kaggle.com/nurulsyazanaramlee/diabetes-prediction

2) Brief description of data

The data is related with health problems like diabetis 


3) What is the target?
 if the person has or nort diabetis


 4) Is this a classification or regression problem?

 classification problem


 5) How many features?


 7 features 1 target

6) How many rows of data.

  768


 7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

 i think has to be some visualisations problems also cleaning exploring and 
 use the best  model to clasificated our model



